### DISCUSSION TITLE
How to implement test to check 'Remember me' functionality using this framework?

### DISCUSSION THREAD

**Discussion by ipetrov1987** [akshayp7/playwright-typescript-playwright-test#67] [Q&A]
Hi,

I want to check 'Remember me' using the following steps:
1. Navigate to login page
2. Login with 'Remember me' enabled
3. Verify that user is logged in and navigated to correct(available only after login) page
4. Close page/browser
5. Open page/browser
6. Navigate to login page
7. Verify that user is logged in and navigated to correct(available only after login) page

Appreciate any help with this.
Thank you in advance!

**Comment by akshayp7** [comment]
Means you want to persist the session after clicking Remeber me??
